genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/v4/logs/JobLogFile.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Perhaps semantic but seems reasonable enough. If we do this the whole service should probably be named `JobFileService` instead of `LogService` for consistency I would think